packages/components/src/components/TokenInput/TokenInput.tsx

```diff
@@ -0,0 +1,109 @@
+import {
+  FormControl,
+  FormLabel,
+  Input,
+  InputGroup,
+  InputProps,
+  InputRightElement,
+  Select,
+  SelectFieldProps,
+} from '@chakra-ui/react';
+import React from 'react';
+import { defaultTokens } from './tokens';
+
+export interface Token {
+  // token sybmol
+  symbol: string;
+  // token contract address
+  address: string;
+}
+
+export interface TokenInputProps {
+  /**
+   * an array of tokens to display
+   */
+  tokens?: Token[];
+  /**
+   * value of the input
+   */
+  value: string;
+  /**
+   * onChange handler
+   */
+  onValueChange: (value: string) => void;
+  /**
+   * label for the input
+   */
+  label?: string;
+  /**
+   * selected token
+   */
+  selectedToken: string;
+  /**
+   * token change hanlder
+   */
+  onTokenChange: (token: string) => void;
+  /**
+   * want to show default tokens
+   */
+  showDefualtTokens?: boolean;
```

Comment:
```suggestion
  showDefaultTokens?: boolean;
```

Reply:
Please also update any references to this in this file or other files

Reply:
Yeah, sure